sourcecode/apis/contentauthor/Dockerfile

```diff
@@ -24,7 +22,15 @@ RUN set -eux; \
         sockets \
         xmlrpc \
         zip \
-    ; \
+    ;
+
+
+FROM php_platform AS php_base
+
+COPY . .
```

Comment:
If you look at the old dockerfile I had optimized the composer install. basically you copy composer.json and composer.lock first then run composer install and at last you copy the source code. This will ensure composer install is only run when dependencies changes and there is no cache

Reply:
Don't think this is a viable approach, as Composer also generates optimised autoload files and such. Leaving this to be looked at in the future.

Reply:
If you actually look at the old Dockerfile of contentauthor (https://github.com/cerpus/Edlib/blob/671de4f67ecd3180d929c94b767711d5fd8ff9df/sourcecode/apis/contentauthor/Dockerfile#L31) it was already solved. I guess this was broken after the refactoring so it would be nice if it could be fixed again.